README.md

```diff
@@ -1,24 +1,235 @@
+Once the download is complete you will be able to access my project locally.
+<!--
+Example commands:
+
+```bash
+$ git clone git@github.com:uchexm/hello-rails-react.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+  ```bash
+- bundle install
+- npm install
+```
+--->
+
+### Usage
+
+  Create Database (Mandatory)
+
+```bash
+rails db:create
+rails db:schema:load
+```
+
+If you have made a migration the run this command
+
+```bash
+rails db:migrate
+rails s
+``` 
+<!--
+Example command:
+
+```sh
+  rails server
+  yarn build --watch
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+
+ ```bash
+rspec
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+Heroku
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Uchechi Nwaka**
+
+- GitHub: [@uchexm](https://github.com/uchexm)
+- Twitter: [@nwakauc](https://twitter.com/nwakauc)
+- LinkedIn: [Uchechi Nwaka](https://linkedin.com/in/uchechi-nwaka-79a21a75/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- Add more endpoints
+- Display Date and Time
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/uchexm/hello-rails-react/issues.)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I ascribe all Glory to the Almighty God.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Rails and React is not soo difficult to learn]**
+
+  - [You need to be conversant with the necessary GEMS]
+
+- **[BUilding Rails API to an existing app]**
+
+  - [Create a new API controller with its route config in routes.rb]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ OPTIONAL] 
This text is provided at part of the README template to guide on adding the MIT license. You can consider deleting it to make your README look clean

Reply:
Fixed.